### DISCUSSION TITLE
Plan for SU2 v8

### DISCUSSION THREAD

**Discussion by pcarruscag** [su2code/SU2#1925] [General]
Hi all, we will have SU2 v8 this year (start thinking about bird names) this is an opportunity to break backward compatibility and thus lose some legacy weight. We will:
- Have a new python wrapper interface after the work of @patelha57 and @aa-g 
- Fix a long-time inconsistency in the SST model
- Remove the legacy output class
- Remove the legacy build scripts
- Try to remove some old python scripts which don't add much to the code

If you have more ideas or concerns post them here.

**Comment by NAnand-TUD** [comment]
How about the iso-thermal Streamwise Periodic flow solver :)

**Comment by bigfooted** [comment]
> start thinking about bird names
https://ebird.org/species/hoopoe

The flamelet solver implementation is also pretty far, I still have to clean up the axisymmetry implementation and fix the conjugate heat transfer implementation.

**Comment by pcarruscag** [comment]
New features are not a problem, que can add them at any point as long as the code is still compatible.

**Comment by WallyMaier** [comment]
We would like to update the version of M++ before v8.
